tap_mailchimp/schemas/campaigns.json

```diff
@@ -453,6 +464,360 @@
+        "object"
+      ],
+      "additionalProperties": false,
+      "properties": {
+        "winning_combination_id": {
+          "type": [
+            "null",
+            "string"
+          ]
+        },
+        "winning_campaign_id": {
+          "type": [
+            "null",
+            "string"
+          ]
+        },
+        "winner_criteria": {
+          "type": [
+            "null",
+            "string"
+          ]
+        },
+        "wait_time": {
+          "type": [
+            "null",
+            "string"
+          ]
+        },
+        "test_size": {
+          "type": [
+            "null",
+            "string"
+          ]
+        },
+        "subject_lines": {
+          "type": [
+            "null",
+            "string"
+          ]
+        },
+        "send_times": {
+          "type": [
+            "null",
+            "string"
+          ]
+        },
+        "from_names": {
+          "type": [
+            "null",
+            "string"
+          ]
+        },
+        "reply_to_addresses": {
+          "type": [
+            "null",
+            "string"
+          ]
+        },
+        "contents": {
+          "type": [
+            "null",
+            "string"
+          ]
+        },
+        "combinations": {
+          "type": [
+            "null",
+            "object"
+          ],
+          "additionalProperties": false,
+          "properties": {
+            "id": {
+              "type": [
+                "null",
+                "string"
+              ]
+            },
+            "subject_line": {
+              "type": [
+                "null",
+                "string"
+              ]
+            },
+            "send_time": {
+              "type": [
+                "null",
+                "string"
+              ]
+            },
+            "from_name": {
+              "type": [
+                "null",
+                "string"
+              ]
+            },
+            "reply_to": {
+              "type": [
+                "null",
+                "string"
+              ]
+            },
+            "content_description": {
+              "type": [
+                "null",
+                "string"
+              ]
+            },
+            "recipients": {
+              "type": [
+                "null",
+                "string"
+              ]
+            }
+          }
+        }
+      }
+    },
+    "rss_opts": {
+      "type": [
+        "null",
+        "object"
+      ],
+      "additionalProperties": false,
+      "properties": {
+        "feed_url": {
+          "type": [
+            "null",
+            "string"
+          ]
+        },
+        "frequency": {
+          "type": [
+            "null",
+            "string"
+          ]
+        },
+        "schedule": {
+          "type": [
+            "null",
+            "object"
+          ],
+          "additionalProperties": false,
+          "properties": {
+            "hour": {
```

Comment:
I see the datatype is an integer for this field.

Reply:
Updated the datatype to an integer.